security-command-center/snippets/src/test/java/vtwo/BigQueryExportIT.java

```diff
@@ -0,0 +1,241 @@
+import com.google.cloud.securitycenter.v2.ListBigQueryExportsRequest;
+import com.google.cloud.securitycenter.v2.OrganizationLocationName;
+import com.google.cloud.securitycenter.v2.SecurityCenterClient;
+import com.google.cloud.securitycenter.v2.SecurityCenterClient.ListBigQueryExportsPagedResponse;
+import com.google.cloud.securitycenter.v2.UpdateBigQueryExportRequest;
+import com.google.common.collect.ImmutableList;
+import com.google.protobuf.FieldMask;
+import java.io.IOException;
+import org.junit.Test;
+import org.mockito.MockedStatic;
+import org.mockito.Mockito;
+import vtwo.bigquery.CreateBigQueryExport;
+import vtwo.bigquery.DeleteBigQueryExport;
+import vtwo.bigquery.GetBigQueryExport;
+import vtwo.bigquery.ListBigQueryExports;
+import vtwo.bigquery.UpdateBigQueryExport;
+
+public class BigQueryExportIT {
+
+  public static final String ORGANIZATION_ID = "test-organization-id";
+  public static final String PROJECT_ID = "test-project-id";
+  private static final String LOCATION = "global";
+  private static final String BQ_DATASET_NAME = "test-dataset-id";
+  private static final String BQ_EXPORT_ID = "test-export-id";
+
+  @Test
+  public void testCreateBigQueryExport() throws IOException {
+    // Mock SecurityCenterClient.
+    SecurityCenterClient client = mock(SecurityCenterClient.class);
+    try (MockedStatic<SecurityCenterClient> clientMock = Mockito.mockStatic(
+        SecurityCenterClient.class)) {
+      clientMock.when(SecurityCenterClient::create).thenReturn(client);
+
+      // Define test data.
+      String filter = "test-filter";
+
+      // Build the parent of the request.
+      OrganizationLocationName organizationName = OrganizationLocationName.of(ORGANIZATION_ID,
+          LOCATION);
+
+      // Build the BigQueryExport response.
+      BigQueryExport expectedExport = BigQueryExport.newBuilder()
+          .setDescription(
+              "Export low and medium findings if the compute resource has an IAM anomalous grant")
+          .setFilter(filter)
+          .setDataset(String.format("projects/%s/datasets/%s", PROJECT_ID, BQ_DATASET_NAME))
+          .build();
+
+      // Build the CreateBigQueryExportRequest request.
+      CreateBigQueryExportRequest expectedRequest = CreateBigQueryExportRequest.newBuilder()
+          .setParent(organizationName.toString())
+          .setBigQueryExport(expectedExport)
+          .setBigQueryExportId(BQ_EXPORT_ID)
+          .build();
+
+      // Mock the createBigQueryExport method to return the expected response.
+      Mockito.when(client.createBigQueryExport(any())).thenReturn(expectedExport);
+
+      // Call the createBigQueryExport method.
+      CreateBigQueryExport.createBigQueryExport(ORGANIZATION_ID, LOCATION, PROJECT_ID, filter,
+          BQ_DATASET_NAME, BQ_EXPORT_ID);
+
+      // Verify that the createBigQueryExport method was called with the expected request.
+      Mockito.verify(client).createBigQueryExport(expectedRequest);
```

Comment:
Please statically import `Mockito.verify()` so you can write this line as `verify(client).createBigQueryExport(expectedRequest);`

Same below!

Reply:
done